spring-boot-modules/spring-boot-3-testcontainers/src/main/java/com/baeldung/testcontainers/jdbc/Hobbit.java

```diff
@@ -0,0 +1,39 @@
+package com.baeldung.testcontainers.jdbc;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity(name = "hobbits")
```

Comment:
Maybe we can remove the name value to avoid confusion for readers/students. 

"hobbits" here as an entity does not add specific value but can raise questions "why "hobbits"and not "hobbit"? where is this value being used..etc..

Reply:
makes sense. removed now 👌